app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cached_data = {}
+
+    def wrapper(*args) -> Callable:
```

Comment:
Wrapper return value from function that was calculated, looks like Any type, not Callable here